derive_builder/src/lib.rs

```diff
@@ -519,15 +519,21 @@
 //! [builder pattern]: https://aturon.github.io/ownership/builders.html
 //! [`derive_builder_core`]: https://crates.io/crates/derive_builder_core
 
-#![cfg_attr(feature = "no_std", no_std)]
+extern crate core;
 
 #[allow(unused_imports)]
 #[macro_use]
 extern crate derive_builder_macro;
+
 #[doc(hidden)]
 pub use derive_builder_macro::*;
 
 #[doc(hidden)]
 pub mod export {
-
+    pub use ::core::clone::Clone;
+    pub use ::core::convert::Into;
+    pub use ::core::default::Default;
+    pub use ::core::marker::PhantomData;
+    pub use ::core::option::Option::{self, Some, None};
+    pub use ::core::result::Result::{self, Ok, Err};
```

Comment:
It doesn't seem to; it looks like the system is smart enough to know that these are actually the same ones as in `::std`, or in `::core`